### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by dherbrich**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)? none

### CODE REVIEW

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+This site gives an overview of the resources created for the convenience feature of kubernetes service account autoconfiguration for AMR authentication and authorization. It will help you understand which resource plays a role in the default configuration and where to look in case issues arise. If you need a deeper understanding of the resources and if you are in need to set up these resources on your own, for example, if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-autoconfiguration.md:3]:
```
If you need a deeper understanding of the resources and if you are in need to set up these resources on your own, for example, if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
```

This sentence is a little awkward. What do you think about simplifying it down to something like: 

```
If you need a deeper understanding of the resources or if you need to set up these resources, head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md). 
```

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+All package-level configurations are prefixed with a component top level key (TLK) and can also be influenced by certain TAP level configurations, which will be mentioned in the separate sections.
 
 
 ## Observer
 
 Observer configuration in the TAP context is prefixed with the TLK `amr.observer`. 
-For authentication and authorization the TAP profiles will influence autconfiguration. Observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
+For authentication and authorization, the TAP profiles will influence autoconfiguration. The observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-autoconfiguration.md:11]: `auth.kubernetes_service_accounts.autoconfigure` would need to be set to false for any profile that is not full or view, right? It doesn't happen automatically does it? If it is not automatic, can you rephrase the last sentence to something like:

```
If it is not being installed in the `full` or `view` profile, `auth.kubernetes_service_accounts.autoconfigure` should be set to false in the `values.yaml` file. 
```

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
 * a `Secret` named `amr-observer-edit-token` of type `kubernetes.io/service-account-token` which will generate a long-lived token for the service account
-* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissons for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
+* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissions for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
 * a `ClusterRoleBinding` named `tanzu:amr:observer:editor` binding the defined role to the service account 
 
 
-
 ## Cloudevent handler
 
 Cloudevent handler configuration in the TAP context can be found under the TLK `amr.cloudevent_handler`. This prefix is not stripped in this case.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-autoconfiguration.md:23]: I believe Nicholas just recently flattened the structure such that this is no longer true. 

Cloudevent handler is now under `metadata_store.cloudevent_handler` as opposed to `metadata_store.amr.cloudevent_handler`. Similarly, GraphQL is now under `metadata_store.graphql` as opposed to `metadata_store.amr.graphql`

File: scst-store/amr/auth-k8s-sa-user-defined.hbs.md [hunk 4]
```diff
@@ -53,9 +53,9 @@ metadata:
     kubernetes.io/service-account.name: amr-observer
 ```
 
-If you saved this to a file `observer-rbac.yaml` you can run `kubectl apply -f observer-rbac.yaml` to set everything up. If you prefer short lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-observer-editor-token -n amr-observer-system` to create a token.
+If you saved this to a file `observer-rbac.yaml` you can run `kubectl apply -f observer-rbac.yaml` to set everything up. If you prefer short-lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-observer-editor-token -n amr-observer-system` to create a token.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:56]: `If you saved this to a file `observer-rbac.yaml` you can run` -> `If you saved this to a file `observer-rbac.yaml`, you can run`

Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:56]: `remove the secret from the file beforehand and after creating the resources run` -> `remove the secret from the file beforehand and after creating the resources, and then run`

File: scst-store/amr/auth-k8s-sa-user-defined.hbs.md [hunk 5]
```diff
@@ -53,9 +53,9 @@ metadata:
...
 
-Please note that if you want to configure custom service accounts even if autoconfiguratin is turned on you will need to modify adjust the naming of the resources.
+Please note that if you want to configure custom service accounts even if autoconfiguration is turned on you will need to adjust the naming of the resources.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:58]: `if you want to configure custom service accounts even if autoconfiguration is turned on you will` -> `if you want to configure custom service accounts even if autoconfiguration is turned on, you will`

File: scst-store/amr/auth-k8s-sa-user-defined.hbs.md [hunk 6]
```diff
@@ -109,4 +109,4 @@ metadata:
 
 If you saved this to a file `graphql-client-rbac.yaml` you can run `kubectl apply -f graphql-client-rbac.yaml` to set everything up. If you prefer short lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-graphql-view-token -n metadata-store` to create a token.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:110]:
```
If you saved this to a file `graphql-client-rbac.yaml` you can run `kubectl apply -f graphql-client-rbac.yaml` to set everything up.
```
-> 
```
If you saved this to a file `graphql-client-rbac.yaml`, you can run `kubectl apply -f graphql-client-rbac.yaml` to set everything up.
```

Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:110]:
```
If you prefer short lived service account tokens, remove the secret from the file beforehand and after creating the resources run
```
-> 
```
If you prefer short lived service account tokens, remove the secret from the file and after creating the resources, run
```
```

File: scst-store/amr/auth-k8s-sa-user-defined.hbs.md [hunk 7]
```diff
@@ -109,4 +109,4 @@ metadata:
...
 
-Please note that if you want to configure custom service accounts even if autoconfiguratin is turned on you will need to modify adjust the naming of the resources.
\ No newline at end of file
+Please note that if you want to configure custom service accounts even if autoconfiguration is turned on you will need to adjust the naming of the resources.
```
Comment (xtreme-debbie-chen) [inline scst-store/amr/auth-k8s-sa-user-defined.hbs.md:112]: `Please note that if you want to configure custom service accounts even if autoconfiguration is turned on you will need to adjust the naming of the resources.` -> `Please note that if you want to configure custom service accounts even if autoconfiguration is turned on, you will need to adjust the naming of the resources.`

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md (hunk 2 above)
Comment (dherbrich) [reply scst-store/amr/auth-k8s-sa-autoconfiguration.md]: when installed as part of TAP, the value is automatically set to false in the PackageInstall resources for amr-observer. So this should be correct as is.

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md [hunk 8]
```diff
@@ -1,24 +1,23 @@
...
+On the package level, if `auth.kubernetes_service_accounts.enable` and `auth.kubernetes_service_accounts.autoconfigure` are true, the observer package will create the following resources to set up authentication automatically in the namespace `amr-observer-system`:
 
 * a `ServiceAccount` named `amr-observer-editor` that observer will use to send requests to the cloudevent handler
 * a `Secret` named `amr-observer-edit-token` of type `kubernetes.io/service-account-token` which will generate a long-lived token for the service account
-* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissons for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
+* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissions for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
 * a `ClusterRoleBinding` named `tanzu:amr:observer:editor` binding the defined role to the service account
```
[the remark this answers is not in the document]
Comment (dherbrich) [reply scst-store/amr/auth-k8s-sa-autoconfiguration.md]: please check if my edits resolve your comment.

File: scst-store/amr/auth-k8s-sa-autoconfiguration.md (hunk 3 above)
Comment (dherbrich) [reply scst-store/amr/auth-k8s-sa-autoconfiguration.md]: i changed this part to reflect these changes. i believe the values file is still not good so i posted to the slack thread again.

Comment (nedenwalker) [reply scst-store/amr/auth-k8s-sa-autoconfiguration.md]: tap-values has been updated to provide amr properties via the `amr` TLK. so, `amr.cloudevent_handler` and `amr.graphql` for the metadata-store package. `amr.observer` also is present for the amr-observer package. when the properties for the metadata-store are constructed to be injected into the package, properties from the `metadata_store` and `amr` TLKs are merged together. At first, with `amr` properties being injected as is, ie (as `amr.cloudevent_handler` and not `cloudevent_handler`)... essentially it functioned that declaring the amr properties within the metadata_store TLK still functioned. But now that amr properties are injected just as `cloudevent_handler`, the values schema should complain. I can make a slack post to make this clearer.
